packages/runner/injection/multi-domain.js

```diff
@@ -38,4 +38,6 @@
 // test out visiting a 3rd domain
 const cyBridgeFrame = window.parent.frames[2]
 
-cyBridgeFrame.__onBeforeAppWindowLoad(window)
+const Cypress = cyBridgeFrame.Cypress
+
+Cypress.action('app:window:before:load', window)
```

Comment:
Yay now this functions like the main injection script! [^1]

[^1]:https://github.com/cypress-io/cypress/blob/feature-multidomain/packages/runner/injection/main.js#L33